lib/WorldState.js

```diff
@@ -79,11 +78,12 @@ class WorldState {
    * @param {string} json The worldstate JSON string
    * @param {Object} [deps] The options object
    */
-  constructor(json, deps = defaultDeps) {
+  constructor(json, lang = 'en_US', deps = defaultDeps) {
```

Comment:
the latter, but make it a json array with the language key as the object key, so accessing french would be 

`worldState.alerts[0].translations['fr_FR']...`